source/lex.tex

```diff
@@ -1878,8 +1901,7 @@
 \nontermdef{user-defined-floating-point-literal}\br
     fractional-constant \opt{exponent-part} ud-suffix\br
     digit-sequence exponent-part ud-suffix\br
-    hexadecimal-prefix hexadecimal-fractional-constant binary-exponent-part ud-suffix\br
-    hexadecimal-prefix hexadecimal-digit-sequence binary-exponent-part ud-suffix
+    hexadecimal-prefix hexadecimal-fractional-constant binary-exponent-part ud-suffix
```

Comment:
This change also needs to be reverted.

Reply:
Done. Sorry, I don't know how I missed it earlier.